mobile/index.html

```diff
@@ -32,95 +32,72 @@ <h1 class="logo">OnionShare</h1>
 
 	<main id="content">
 
-		<section class="intro homeIntro  wrapper">
+		<section class="intro homeIntro mobileIntro wrapper">
 			<div>
 				<!-- <div class="icon large onionshare"></div> -->
-				<h2>OnionShare Mobile is Ready for Testing!</h2>
-				<p>The OnionShare teams working with Tor Project, Calyx Institute and Guardian Project, 
-					have created the official OnionShare mobile apps for Android and iOS.
-					The apps are now in public beta testing phase, ready for feedback on stability, 
-					usability and privacy controls.
-				</p>
-				<br/><br/>
-				<p class="warning">
-					WARNING: These apps are not yet audited, and in early alpha/beta state. 
-					Please only use for testing.
-				</p>
-				<p class="warning">
-					Report any issues to 
-					<a href="mailto:support@guardianproject.info">Guardian Project Support Helpdesk</a>
-					<br/>
-					or Github on <a href="https://github.com/onionshare">the appropriate Github project</a>
-				</p> 
+				<h1>OnionShare Mobile Beta</h1>
```

Comment:
There should be only one h1 in the page ideally. I agree that this is what should be the h1. Maybe we can update https://github.com/onionshare/onionshare-website/blob/df63c9b599d61b3a94a1a374a119dcfda6d93e48/mobile/index.html#L22 to be some other element since that doesn't really need to be h1